src/chrome/content/rules/ServiceNow.com.xml

```diff
@@ -55,19 +54,16 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="servicenow.com" />
+	<target host="www.servicenow.com" />
 	<target host="community.servicenow.com" />
 	<target host="developer.servicenow.com" />
+	<target host="docs.servicenow.com" />
 	<target host="express.servicenow.com" />
+	<target host="knowledge.servicenow.com" />
```

Comment:
@J0WI can you please provide examples of https and http that aren't the same?  I cannot find any.  I removed the rule from knowledge to community, but http://knowledge is redirected to https://knowledge by their server.